datasets/ziekte_plagen_exoten_groen/japanseduizendknoopInspecties/v1.0.0.json

```
@@ -33,214 +40,223 @@
+                "description": "Dit het oppervlakte van de laatste gemeten omvang van de groeiplaats van de Aziatische duizendknoop"
             },
             "vrijgegeven": {
                 "type": "string",
-                "description": "Beschrijving van het attribuut vrijgegeven."
+                "description": "Deze groeiplaats is succesvol verwijderd en er vind geen hergroei plaats"
             },
             "melding": {
                 "type": "string",
-                "description": "Beschrijving van het attribuut melding."
+                "description": "Dit is waar zich de Aziatische duizenknoop zich volgens de melder bevindt, deze dient geinspecteerd te worden"
             },
             "meldingStatus": {
                 "type": "string",
-                "description": "Beschrijving van het attribuut meldingStatus."
+                "description": "Dit een weergave van de fase of staat waarin de melding zich nu bevindt, of deze nog geinspecteerd moet worden en/of al bestreden wordt"
             },
             "stengelsM2": {
                 "type": "integer",
                 "description": "Beschrijving van het attribuut stengelsM2."
             },
             "maxStengelsDoorsnede": {
                 "type": "integer",
-                "description": "Beschrijving van het attribuut maxStengelsDoorsnede."
+                "description": "Hierin wordt de maximale stengeldikte van de plant weergegeven, wat iets zegt of de ouderdom van de plant"
             },
             "soortnaam": {
                 "type": "string",
-                "description": "Beschrijving van het attribuut soortnaam."
+                "description": "Hier wordt aangegeven welke soort Aziatische duizendknoop is waargenomen tijdens de inspectie, zijnde Japanse, Sachalinse, Boheemse, bastaardduizenknoop of Afghaanse"
             },
-            "percBerm": {
+            "percBebouGevelOfMuur": {
                 "type": "integer",
-                "description": "Beschrijving van het attribuut percBerm."
+                "description": "Percentage Aziatische duizendknoop dat bij gebouwen in gevel of muur groeit"
             },
-            "percPercBermBrug": {
+            "percBebouGeveltuinLosseBakken": {
                 "type": "integer",
-                "description": "Beschrijving van het attribuut percPercBermBrug."
+                "description": "Percentage Aziatische duizendknoop dat bij gebouwen in geveltuinen (losse bakken) groeit"
             },
-            "percBosplantsoen": {
+            "percBebouGeveltuinVast": {
                 "type": "integer",
-                "description": "Beschrijving van het attribuut percBosplantsoen."
+                "description": "Percentage Aziatische duizendknoop dat bij gebouwen in geveltuinen (vaste bakken) groeit"
             },
-            "percBraakliggend": {
+            "percBebouKruipruimte": {
```

Wat is Bebou?
Afkortingen in veldnamen zijn niet toegestaan. En veldnamen moeten begrijpelijk zijn.
 ```suggestion
            "percInKruipruimte": {
```

Idem in velden hierboven